packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -182,7 +184,7 @@ class Checkbox extends StatefulWidget {
   final bool autofocus;
 
   /// The width of a checkbox widget.
-  static const double width = 18.0;
+  static const double width;
```

Comment:
This can't be a `static const` anymore, it will need to be `final`

Reply:
how to contribute to this...

Reply:
I added you to my repo, you can push to `lukepighetti:patch-2`